azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.resources.fluentcore.collection;
+
+public interface ResourcesInGroup<T, DefinitionT> extends
+    SupportsListing<T>,
+    SupportsCreating<DefinitionT>,
+    SupportsDeleting {}
```

Comment:
besides, it won't be the existing SupportsGetting, because that one assumes an ID. What we will need is a new interface: SupportsGettingByName, to make it explicitly expect a name, not an id (.getByName())